lib/grape/idempotency.rb

```diff
@@ -143,12 +159,12 @@ def store_error_request(idempotency_key, path, params, status, request_id, error
           }
         }.to_json
 
-        result = storage.set(error_key(idempotency_key), body, ex: 30, nx: true)
+        result = storage.set(error_key(idempotency_key), body, ex: 30, nx: false)
 
         if !result
-          return store_error_request(random_idempotency_key, path, params, status, request_id, error)
+          store_error_request(random_idempotency_key, path, params, status, request_id, error)
```

Comment:
As we have change the `nx` property to false, the result will not be `true` or `false` anymore. It will always set the value in the storage, so I think this check is not needed anymore. What do you think?

Reply:
true, in fact I leave a non existing method call `random_idempotency_key` that I removed and it's never reaching this part